play.js

```diff
@@ -1353,56 +1353,45 @@ function fetchAndInitLocations(lang, game) {
-        }
-        return response.json();
+  const fileName = lang === 'en' ? 'config' : lang;
+  return fetchNewest(`locations/${game}/${fileName}.json`)
+    .then(response => {
+      return response.ok ? response.json() : Promise.reject();
     })
     .then(jsonResponse => {
-        if (!jsonResponse) {
-          resolve();
-          return;
-        }
-        gameLocalizedLocationUrlRoots[game] = jsonResponse.urlRoot;
-        gameLocalizedMapLocations[game] = {};
-        const langMapLocations = jsonResponse.mapLocations;
-        massageMapLocations(langMapLocations, jsonResponse.locationUrlTitles || null);
-        Object.keys(gameMapLocations[game]).forEach(function (mapId) {
-          const mapLocation = langMapLocations[mapId];
-          const defaultMapLocation = gameMapLocations[game][mapId];
-          if (mapLocation) {
-            gameLocalizedMapLocations[game][mapId] = mapLocation;
-            if (Array.isArray(defaultMapLocation) && Array.isArray(mapLocation) && defaultMapLocation.length === mapLocation.length) {
-              for (let l in defaultMapLocation) {
-                if (defaultMapLocation[l].hasOwnProperty('coords'))
-                  mapLocation[l].coords = defaultMapLocation[l].coords;
-              }
+      gameLocalizedLocationUrlRoots[game] = jsonResponse.urlRoot;
+      gameLocalizedMapLocations[game] = {};
+      const langMapLocations = jsonResponse.mapLocations;
+      massageMapLocations(langMapLocations, jsonResponse.locationUrlTitles || null);
+      Object.keys(gameMapLocations[game]).forEach(function (mapId) {
+        const mapLocation = langMapLocations[mapId];
+        const defaultMapLocation = gameMapLocations[game][mapId];
+        if (mapLocation) {
+          gameLocalizedMapLocations[game][mapId] = mapLocation;
+          if (Array.isArray(defaultMapLocation) && Array.isArray(mapLocation) && defaultMapLocation.length === mapLocation.length) {
+            for (let l in defaultMapLocation) {
+              if (defaultMapLocation[l].hasOwnProperty('coords'))
+                mapLocation[l].coords = defaultMapLocation[l].coords;
             }
-          } else
-            gameLocalizedMapLocations[game][mapId] = defaultMapLocation;
-        });
-        if (game === gameId) {
-          localizedLocationUrlRoot = gameLocalizedLocationUrlRoots[game];
-          localizedMapLocations = gameLocalizedMapLocations[game];
-        }
-        initLocalizedLocations(game);
-        resolve();
+          }
+        } else
+          gameLocalizedMapLocations[game][mapId] = defaultMapLocation;
+      });
+      if (game === gameId) {
+        localizedLocationUrlRoot = gameLocalizedLocationUrlRoots[game];
+        localizedMapLocations = gameLocalizedMapLocations[game];
+      }
+      initLocalizedLocations(game);
     })
     .catch(_err => { // Assume map location localizations for this language don't exist
```

Comment:
Is this comment still accurate?

Reply:
Yes, it was kept there and removed elsewhere, since there is now the convergence where it's assumed the language does not have a location localization.